examples/platform/silabs/efr32/rs911x/wifisdk.gni

```diff
@@ -0,0 +1,41 @@
+import("//build_overrides/chip.gni")
+
+examples_plat_dir = "${chip_root}/examples/platform/silabs/efr32"
+wifi_sdk_dir = "${chip_root}/src/platform/silabs/efr32/wifi"
+wiseconnect_sdk_root = "${chip_root}/third_party/silabs/wiseconnect-wifi-bt-sdk"
+#sl_wifi_sdk_root = "${chip_root}/third_party/silabs/sl_net/components"
+sl_wifi_sdk_root = "${chip_root}/third_party/silabs/wifi_sdk-v3.0.3/components"
```

Comment:
This path seems to be incorrect

Reply:
This file is deleted as we will not use for soc.